src/Modules/CurrentMapModule/Services/CurrentMapService.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using EvoSC.Common.Interfaces;
+using EvoSC.Common.Interfaces.Database.Repository;
+using EvoSC.Common.Services.Attributes;
+using EvoSC.Common.Services.Models;
+using EvoSC.Manialinks.Interfaces;
+using EvoSC.Modules.Official.CurrentMapModule.Interfaces;
+using Microsoft.Extensions.Logging;
+using GbxRemoteNet.Events;
+using ISO3166;
+
+namespace EvoSC.Modules.Official.CurrentMapModule.Services;
+
+[Service(LifeStyle = ServiceLifeStyle.Transient)]
+public class CurrentMapService : ICurrentMapService
+{
+    private readonly ILogger<CurrentMapService> _logger;
+    private readonly IManialinkManager _manialinkManager;
+    private readonly IMapRepository _mapRepository;
+    private readonly IServerClient _client;
+
+    public CurrentMapService(IManialinkManager manialinkManager,
+        ILogger<CurrentMapService> logger,
+        IMapRepository mapRepository, IServerClient client)
+    {
+        _logger = logger;
+        _manialinkManager = manialinkManager;
+        _mapRepository = mapRepository;
+        _client = client;
+    }
+
+    private static string GetCountry(string Zone)

```

Comment:
Could you change the order of accessor methods from private -> public to public -> private?